crates/bin/src/ui.rs

```diff
@@ -35,10 +35,7 @@ pub async fn confirm() -> Result<(), BinstallError> {
 
             match input.as_str().trim() {
                 "yes" | "y" | "YES" | "Y" => break true,
-                "no" | "n" | "NO" | "N" | "" => break false,
-                _ => {
-                    input.clear();
-                    continue;
+                _ => => break false,
```

Comment:
That's a bit too much change, you just need to remove the `""` from
the `break false` branch and move it to `break true` branch.